copier.yaml

```diff
@@ -3,6 +3,7 @@ _subdirectory: template
 full_project_name:
   type: str
   help: "Full project name (go module path) ex: github.com/catalystsquad/example-app"
+  default: "github.com/catalystcommunity/sprinkles"
```

Comment:
imo, I don't think the default should be a real project

Reply:
Changing it to example-app like the help